boost-maven-plugin/src/main/java/io/openliberty/boost/LibertyBoostMojo.java

```diff
@@ -231,29 +256,16 @@ private ExecutionEnvironment getExecutionEnvironment() {
      * @return
      */
     private List<String> getBoosterFeatureNames(List<BoosterPackConfigurator> boosterConfigurators) {
-			List<String> featureStrings = new ArrayList<String>();
-			for (BoosterPackConfigurator bpconfig : boosterConfigurators){
-				featureStrings.add(bpconfig.getFeatureString());
-			}
-			
-			return featureStrings;
+		List<String> featureStrings = new ArrayList<String>();
+		for (BoosterPackConfigurator bpconfig : boosterConfigurators){
+			featureStrings.add(bpconfig.getFeatureString());
 		}
-
-	private List<String> findJ2EEAppFeatureDependencies(MavenProject project) {
 		
-    	List<String> listOfDependencies = new ArrayList<String>();
-		getLog().debug("first lets see what dependencies we find");
-		for (Artifact artifact : mavenProject.getArtifacts()) {
-			getLog().debug("found this dependency, adding as a string -> " + artifact.getArtifactId());
-			listOfDependencies.add(artifact.getArtifactId());
-		}
-    	
-    	return listOfDependencies;
+		return featureStrings;
```

Comment:
Did you mean to remove the findJ2EEAppFeatureDependencies() method?

Reply:
The method wasn't being used. Not sure if it was necessary.